fearless/Modules/Staking/Stories/Model/Slide.swift

```diff
@@ -0,0 +1,18 @@
+struct Slide {
```

Comment:
Looks like very general. I suggest to either change to StorySlide or put this defintion inside Story struct

Reply:
👍